ArduCopter/system.cpp

```diff
@@ -480,10 +480,47 @@ const char* Copter::get_frame_string() const
             return "DODECA_HEXA";
         case AP_Motors::MOTOR_FRAME_DECA:
             return "DECA";
+        case AP_Motors::MOTOR_FRAME_SCRIPTING_MATRIX:
+            return "SCRIPTING_MATRIX";
         case AP_Motors::MOTOR_FRAME_UNDEFINED:
-        default:
             return "UNKNOWN";
     }
+    // unknown frame
+    return "UNKNOWN";
+}
+
+// return MAV_TYPE corresponding to frame class
+MAV_TYPE Copter::get_frame_mav_type()
```

Comment:
I think this needs a rebase, I just moved it to motors in https://github.com/ArduPilot/ardupilot/commit/d0e1e85b9e43e181b597d94f8cf9de5515b079e5#diff-019889aa8f604ba16f535c5d61b768827e08bf3d1934b5e930f74b7645ae1405

Reply:
that's confusing... I though I rebased this morning on latest master